pallets/asset-manager/src/weights.rs

```diff
@@ -53,45 +53,62 @@ use sp_std::marker::PhantomData;
 /// Weight functions needed for pallet_asset_manager.
 pub trait WeightInfo {
 	fn register_asset() -> Weight;
-	fn set_asset_units_per_second() -> Weight;
-	fn change_existing_asset_type() -> Weight;
+	fn set_asset_units_per_second(x: u32) -> Weight;
+	fn change_existing_asset_type(x: u32) -> Weight;
+	fn remove_supported_asset(x: u32) -> Weight;
 }
 
 /// Weights for pallet_asset_manager using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	fn register_asset() -> Weight {
-		(52_375_000 as Weight)
+		(42_957_000 as Weight)
```

Comment:
Were these results run on a different machine? That's a fairly big decrease given that the fn itself didn't change. It could also be that upstream code became more efficient...?

Reply:
It might be best to just add the new benchmark fns to this file and ignore any "noise" in measurements

Reply:
all of them were run in the target machine. But I will leave as ir was